### DISCUSSION TITLE
Popup on iOS doesn't respect VerticalStackLayout padding at the bottom of the layout

### DISCUSSION THREAD

**Discussion by JRosanowski** [CommunityToolkit/Maui#2156] [General]
Has anyone else come across this? I've got a VerticalStackLayout with padding of 16, and buttons at the bottom of the layout. On Android there's padding below the buttons. On iOS there isn't.

**Comment by stephenquan** [comment]
Did you try Grid?

**Reply by JRosanowski** [reply]
I seem to get more problems with grid. The only solution I've found is to wrap the VerticalStackLayout in another and set the inner one to have a margin instead of padding.

**Comment by CliffAgius** [comment]
I have converted this to an Issue #2167 and will close this discussion.
